src/components/AuthButtons.tsx

```diff
@@ -0,0 +1,45 @@
+import { Box, Button } from "@mui/material";
+import { useNavigate } from "react-router-dom";
+
+type AuthButtonsProps = {
```

Comment:
Somewhere you use type for defining props and somewhere interfaces.

For props handling common practise is to use interface. It's not bad at all using type, but then keep consistency and use it everywhere for props.